### PR TITLE
Repair Documentation Links

### PR DESCRIPTION
**Pull request by jonreding2010**
Updated documentation links in _sidebar file.
Added documentation files for the ones that were missing.

There are doc files that still need to be finished. These are laid out in issue #283

### CODE REVIEW

File: docs/JMAQS_1/appium/AppiumBaseTest.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Available calls
+[GetMobileDevice](#GetMobileDevice)  
+[BeforeLoggingTeardown](#BeforeLoggingTeardown)  
+[CreateNewTestObject](#CreateNewTestObject)  
+
+## GetMobileDevice
+This method gets the default Appium/Mobile driver. 
+```java
+protected virtual AppiumDriver<IWebElement> GetMobileDevice()
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumBaseTest.md:14]: C# syntax

File: docs/JMAQS_1/appium/AppiumBaseTest.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## BeforeLoggingTeardown
+Takes a screen shot if needed and tear down the appium driver. It is called during teardown.
+```java
+this.BeforeLoggingTeardown(resultType);
+```
+
+## CreateNewTestObject
+This method creates a new Appium test object based on the mobile device.
+```java
+ protected override void CreateNewTestObject()
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumBaseTest.md:29]: C# syntax

File: docs/JMAQS_1/webservice/WebServiceDriverManager.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# <img src="resources/maqslogo.ico" height="32" width="32"> Web Service Driver Manager
+
+## Overview
+The WebService Driver Manager has overreach of the Base Driver Manager.
+
+[OverrideDriver](#OverrideDriver)
+[GetWebSeriveDriver](#GetWebSeriveDriver)
```
Comment (jason-edstrom) [inline docs/JMAQS_1/webservice/WebServiceDriverManager.md:7]: Typo

File: docs/JMAQS_1/base/BaseFeatures.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# <img src="resources/maqslogo.ico" height="32" width="32"> Base Basics
+
+## Overview
+MAQS provides support for testing, base is the foundation for this support.  
+
+## BaseTest
+ Base for tests without a defined system under test
+ ```java
+  MaqsBase basetest = new MaqsBase();
```
Comment (jason-edstrom) [inline docs/JMAQS_1/base/BaseFeatures.md:9]: C# syntax

File: docs/JMAQS_1/base/BaseFeatures.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+## TestObject
+Base test context data
+```java
+BaseTestObject baseTestObject = new BaseTestObject(new ConsoleLogger(), string.Empty);
+```
+
+## DriverManager
+Base driver manager object
+```java
+DriverManager managerToKeep = GetManager();
```
Comment (jason-edstrom) [inline docs/JMAQS_1/base/BaseFeatures.md:24]: C# syntax

File: docs/JMAQS_1/appium/AppiumFAQ.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+```
+- Natively supported browsers are:  Firefox | Chrome | Edge | PhantomJS | Internet Explorer. 
+Find the configuration of browsers within the app.config file and define one and only one.
+
+- App.Config: Want to make changes to the Platform, Device, Logging, Remote Integration Hubs, Wait times, Creating Logs, Log File Format and Log folder location. Look to the App.Config within Tests
+
+### How do I fix common user errors?
+
+### My code can't find the config class  
+- Make sure you've imported the Full namespace such as Magenic.Maqs.Utilities.Helper.Config
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumFAQ.md:17]: C# reference

File: docs/JMAQS_1/appium/AppiumFAQ.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+### Test can't find the page element for a test, errors indicate something is wrong with Selenium  
+- Kill the existing Chrome processes spawned and orphaned from the test kick off then update Browser and start fresh
+### Test fails on driver, such as Chrome  
+- Update the Chrome browser  (In Chrome click the HotDog icon (formerly known as Wrench) > help > about Google Chrome > Update , the key is that the Chrome browser version and the ChromeDriver versions must be compatible with each other.
+
+## Build Solution Fails
+- If NuGet packages are missing, Right-click the solution and Enable NuGet Package restore (this requires VPN or LAN connection) Unless an off network storage location or shared folder, was setup to hold the NuGet packages.  Visual Studio allows you to configure additional NuGet locations (see below)
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumFAQ.md:24]: C# reference

File: docs/JMAQS_1/appium/AppiumBaseTest.md (hunk 1 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumBaseTest.md]: Syntax has been changed.

File: docs/JMAQS_1/appium/AppiumBaseTest.md (hunk 2 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumBaseTest.md]: Syntax has been changed.

File: docs/JMAQS_1/webservice/WebServiceDriverManager.md (hunk 3 above)
Comment (jonreding2010) [reply docs/JMAQS_1/webservice/WebServiceDriverManager.md]: type has been fixed

File: docs/JMAQS_1/base/BaseFeatures.md (hunk 4 above)
Comment (jonreding2010) [reply docs/JMAQS_1/base/BaseFeatures.md]: Syntax has been fixed

File: docs/JMAQS_1/base/BaseFeatures.md (hunk 5 above)
Comment (jonreding2010) [reply docs/JMAQS_1/base/BaseFeatures.md]: syntax has been fixed

File: docs/JMAQS_1/appium/AppiumFAQ.md (hunk 6 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumFAQ.md]: Reference has been resolved

File: docs/JMAQS_1/appium/AppiumFAQ.md (hunk 7 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumFAQ.md]: Added  a TODO for future reference. This is dependent on the Build Documentation

**Comment by jonreding2010** [comment]
@jason-edstrom I've resolved the requested changes and added more documentation but it's not allowing the build to continue. When you have time, can you take a look?

**Comment by jason-edstrom** [comment]
I believe that the docs folder is excluded from the build trigger since it is just a folder of markdown files.

File: docs/JMAQS_1/utilties/Logger.md [hunk 8]
```diff
@@ -1,5 +1,4 @@
-# <img src="resources/maqslogo.ico" height="32" width="32"> Logger
-
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Logger
 
 ## Overview
```
Comment (jason-edstrom) [inline docs/JMAQS_1/utilties/Logger.md:3]: Multiple references in this file to csharp things.  DLL in copy, code blocks marked as C#.

File: docs/JMAQS_1/selenium/SeleniumConfig.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# <img src="resources/maqslogo.ico" height="32" width="32"> SeleniumConfig
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> SeleniumConfig
```
Comment (jason-edstrom) [inline docs/JMAQS_1/selenium/SeleniumConfig.md:1]: Remove appsettings.json section

File: docs/JMAQS_1/base/DriverManager.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Driver Manager
+
+## Overview
+The DriverManager has overreach of te BaseTestObject and the BaseDriver.
+
+[GetLogger](#GetLogger)
+[GetBaseDriver](#GetBaseDriver) 
+[SetBaseDriver](#SetBaseDriver)  
+[IsDriverInitialized](#IsDriverInitialized)
+[GetBase](#GetBase)
```
Comment (jason-edstrom) [inline docs/JMAQS_1/base/DriverManager.md:10]: Check the rendering on this section.  Not every link is on a new line.

File: docs/JMAQS_1/appium/AppiumConfig.md [hunk 11]
```diff
@@ -1,243 +1,85 @@
...
-<br>The AppiumCapsMaqs configuration section contains the following Keys:  
-* ***Username*** : Name of the user
-* ***AccessKey*** : The key to access the testing device
-* ***DeviceOrientation*** : The screen orientation of the device (portrait or auto-rotate)
-* ***BrowserName*** : Name of the Browser being used
-
-## Sample config files
-#### config.xml
+# Sample config files
+## App.config
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumConfig.md:73]: config.xml

File: docs/JMAQS_1/appium/AppiumConfig.md [hunk 12]
```diff
@@ -261,30 +103,75 @@ These are key value pairs that get added to the remote web driver's desired capa
...
+    <add key="accessKey" value="7e0592a4-16de-4c6b-9b87-ee61aa43ceac" />
+    <add key="deviceName" value="Android Emulator" />
+    <add key="deviceOrientation" value="portrait" />
+    <add key="browserName" value="Chrome" />
+  </AppiumCapsMaqs>
 </configuration>
 ```
-### appsettings.json
+
+## appsettings.json
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumConfig.md:157]: remove json section

File: docs/JMAQS_1/appium/AppiumDriverFactory.md [hunk 13]
```diff
@@ -1,114 +1,59 @@
...
-Package: com.magenic.jmaqs.appium;
-Assembly: import com.magenic.jmaqs.appium.AppiumDriverFactory
-
-## Syntax
-Java
-```java
-public class AppiumDriverFactory
-```
+## Overview
+Takes care of the base test context data.
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumDriverFactory.md:4]: Not an accurate overview

File: docs/JMAQS_1/appium/AppiumFAQ.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Appium FAQ
+
+## What Browsers can I use?
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumFAQ.md:3]: Looks like this is all from the selenium FAQ

File: docs/JMAQS_1/appium/AppiumFeatures.md [hunk 15]
```diff
@@ -1,12 +1,68 @@
...
-[BaseAppiumTest](#BaseAppiumTest) - Functionality for Appium and Mobile Drivers  
-[MobileDriverManager](#MobileDriverManager) - Mobile Driver Manager
\ No newline at end of file
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Appium Features
+
+## Overview
+JMAQS provides support for testing mobile applictions.  	
+
+## BaseTest
+Base Appium Test is an abstract test class you can extend.  Extending the class allows you to automatically use MAQS's web service testing capabilities.
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumFeatures.md:7]: calls out MAQS and webservice testing

File: docs/JMAQS_1/appium/AppiumTestObject.md [hunk 16]
```diff
@@ -1,84 +1,26 @@
...
-```
-Package: com.magenic.jmaqs.appium;  
-Assembly: import com.magenic.jmaqs.appium.AppiumTestObject
-
-## Syntax
-Java
-```java
-public class AppiumTestObject
-```
+## Overview
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumTestObject.md:3]: Update overview

File: docs/JMAQS_1/appium/AppiumTestObject.md [hunk 17]
```diff
@@ -1,84 +1,26 @@
...
-
-##  AppiumTestObject
-Instantiates a new Appium test object.
-#### Written as
- ```java
-public AppiumTestObject(AppiumDriver<WebElement> appiumDriver, Logger logger, String fullyQualifiedTestName)
-```
-#### Example
+## GetAppiumDriver
+Gets the web service driver
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumTestObject.md:11]: webservices reference

File: docs/JMAQS_1/appium/AppiumTestObject.md [hunk 18]
```diff
@@ -1,84 +1,26 @@
...
-AppiumTestObject appiumTestObject = new AppiumTestObject(defaultMobileDriver,
-new ConsoleLogger(), method.getName());
+AppiumDriver<WebElement> driver = this.AppiumManager.getAppiumDriver();
 ```
 
-##  GetAppiumDriver
-Gets the Appium Driver
-#### Written as
+## SetAppiumDriver
+Sets the web service driver
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumTestObject.md:17]: webservices reference

File: docs/JMAQS_1/appium/AppiumTestObject.md [hunk 19]
```diff
@@ -1,84 +1,26 @@
...
-##### With a Web Element Parameter
-```java
-AppiumTestObject appiumTestObject = new AppiumTestObject(defaultMobileDriver,
-    consoleLogger,testName);
 appiumTestObject.setAppiumDriver(AppiumDriverFactory.getDefaultMobileDriver());
 ```
-#### With a Supplier Parameter
+
+## GetAppiumManager
+Gets the web service driver manager
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/AppiumTestObject.md:23]: webservices reference

File: docs/JMAQS_1/appium/MobileDriverManager.md [hunk 20]
```diff
@@ -1,64 +1,35 @@
...
-
-## Properties
-BaseDriver - (Inherited from DriverManager.)  
-GetDriver - (Inherited from DriverManager.)  
-Log	- (Inherited from DriverManager.)
-
-## Methods
-[GetMobileDriver](#GetMobileDriver)  - gets the mobile driver
-[Close](#Close) - closes the driver
+## Overview
```
Comment (jason-edstrom) [inline docs/JMAQS_1/appium/MobileDriverManager.md:3]: this reads weird.

File: docs/JMAQS_1/appium/AppiumConfig.md (hunk 11 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumConfig.md]: changed to config.xml

File: docs/JMAQS_1/appium/AppiumFAQ.md (hunk 14 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumFAQ.md:3]: Left this page blank and added it to Update Documentation Issue #283

File: docs/JMAQS_1/appium/AppiumTestObject.md (hunk 16 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumTestObject.md:17]: overview has been updated

File: docs/JMAQS_1/appium/AppiumTestObject.md (hunk 18 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumTestObject.md]: webservices reference has been deleted and replaced with Appium

File: docs/JMAQS_1/appium/AppiumTestObject.md (hunk 17 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumTestObject.md]: replaced reference

File: docs/JMAQS_1/appium/AppiumFeatures.md (hunk 15 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumFeatures.md]: added j to MAQS

File: docs/JMAQS_1/appium/AppiumDriverFactory.md (hunk 13 above)
Comment (jonreding2010) [reply docs/JMAQS_1/appium/AppiumDriverFactory.md:4]: @jason-edstrom  review needed for updated overview

File: docs/JMAQS_1/utilties/Logger.md (hunk 8 above)
Comment (jonreding2010) [reply docs/JMAQS_1/utilties/Logger.md:3]: @jason-edstrom  not sure exactly what looks like C# stuff but made some changes. Review needed to confirm if changes are valid.

Comment (jason-edstrom) [reply docs/JMAQS_1/utilties/Logger.md:3]: You hit 99% of my concerns for this file.  For the sake of splitting hairs, consider changing `this.log` to `this.getLogger`.  More accurate to the code base. :)

**Pull request merged**